.circleci/config.yml

```diff
@@ -228,13 +219,6 @@ jobs:
                     curl https://nixos.org/nix/install | sh
                     . ~/.nix-profile/etc/profile.d/nix.sh
                     nix-env -iA cachix -f https://cachix.org/api/v1/install
-            - run:
```

Comment:
we still need kademlia for CI I thought, isn't this going to make the builds slow until my removal lands?

Reply:
This is specific to the mac build, so it shouldn't break CI tests.